jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -1414,6 +1414,9 @@
       ))
   (:wait-interpolation-until-grasp
     (arm)
+    (when (send self :simulation-modep)
+      (return-from :wait-interpolation-until-grasp (send self :wait-interpolation))
+      )
```

Comment:
the problem is that `wait-interpolation-until-grasp` finishes to quickly in simulation, right?